bug-bounty-preview-scopes.md

```diff
@@ -0,0 +1,33 @@
+# .NET Bounty Scope
```

Comment:
The bug bounty seems to be scoped to security features. Should the title reflect that?

Reply:
Legal :) But yea, the bounty only covers things that would create an MSRC, that's just the rules.